infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```
@@ -332,9 +332,9 @@ var varSubnetProperties = [for subnet in varSubnetMap: {
   }
 }]
 
-var varVpnGwConfig = ((!empty(parVpnGatewayConfig)) ? parVpnGatewayConfig : json('{"name": "noconfigVpn"}'))
+var varVpnGwConfig = ((parVpnGatewayEnabled) ? parVpnGatewayConfig : json('{"name": "noconfigVpn"}'))
```

```suggestion
var varVpnGwConfig = ((parVpnGatewayEnabled) &&(!empty(parVpnGatewayConfig)) ? parVpnGatewayConfig : json('{"name": "noconfigVpn"}'))
```

should we keep this logic to check the block isnt empty? i've added it as an AND but do we think it should be an OR?

---

Good point, I think it makes sense as you recommended.